### PR TITLE
Cleaned the tutorial for learning mode in readme

### PR DESCRIPTION
**Pull request by CleanMachine1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,11 @@ make
 # Optional however crucial
 sudo make install # This moves aptpac to /usr/local/bin so it can be run easily
 ```
-if you wan't to enable 'learning mode' (print the commands being run), add `-DLEARN=1` before the `..` in the `cmake` line.
+
+<details>
+<summary><b>Learning Mode</b></summary>
+if you want to enable 'learning mode' which prints the commands being run, add  `-DLEARN=1` before the .. in the cmake line during installation.
+</details>
```
Comment (Itai-Nelken) [inline README.md:65]:
```suggestion

<details>
<summary><b>Learning Mode</b></summary>
if you want to enable 'learning mode' which prints the commands being run, add  `-DLEARN=1` before the .. in the `cmake` line during installation so it looks like this:
```bash
cmake -DLEARN=1 -DCALLCOMMAND="aptpac" -DCMAKE_BUILD_TYPE=Release ..
```
</details>
```

Comment (CleanMachine1) [reply README.md]: did I add it or cancel I can't tell

**Comment by Itai-Nelken** [comment]
removing that as its broken for github pages...

**Comment by Itai-Nelken** [comment]
flameshot_screenshot

**Pull request merged**